pcsx2/SPU2/Reverb.cpp

```diff
@@ -140,14 +142,24 @@ StereoOut32 __forceinline V_Core::ReverbUpsample(bool phase)
 	rs >>= 14;
 	rs = std::clamp<s32>(rs, INT16_MIN, INT16_MAX);
 
-	return StereoOut32(ls, rs);
+	return {ls, rs};
 }
 
-/////////////////////////////////////////////////////////////////////////////////////////
+__forceinline s32 V_Core::RevbGetIndexer(s32 offset)
+{
+	u32 start = EffectsStartA & 0x3f'ffff;
+	u32 end = (EffectsEndA & 0x3f'ffff) | 0xffff;
+
+	u32 x = ((Cycles >> 1) + offset) % ((end - start) + 1);
+
+	x += start;
+
+	return x & 0xf'ffff;
```

Comment:
Shouldn't that be 3fffff?

Reply:
It's the final mask before directly indexing into _spu2mem so i want to mask it to fit within the memory size.

Reply:
fair enough :)